src/triage/util/introspection.py

```diff
@@ -0,0 +1,27 @@
+import inspect
+
+
+def classpath(klass):
+    """Return the full class path
+
+    Args:
+        klass (class): A class
+    """
+    return f"{klass.__module__}.{klass.__name__}"
+
+
+def bind_kwargs(callable, **kwargs):
+    """Bind keyword arguments to a callable and return as a dictionary
+
+    Args:
+        callable (callable): any callable
+        **kwargs: keyword arguments to bind
+
+    Returns: (dict)
+    """
+    call_signature = inspect.signature(callable).bind_partial(**kwargs).arguments
```

Comment:
FYI, `callable` is a built-in function, (so you _might_ actually want to name this parameter `kallable` or something).